### DISCUSSION TITLE
Support for OCaml debugger

### DISCUSSION THREAD

**Discussion by villesau** [giraud/reasonml-idea-plugin#128] [Feature request]
plugin version: v0.69

### Description

In order to fix anything in [Flow](https://github.com/facebook/flow), debugger is a must. To me looks like this is the only considerable IntelliJ plugin for OCaml. Do you have plans to implement debugger for OCaml?

Some pointers:
https://caml.inria.fr/pub/docs/manual-ocaml/debugger.html
vs code adapter: https://github.com/hackwaly/ocamlearlybird
vs code plugin: https://github.com/hackwaly/vscode-ocaml-debugger

**Comment by giraud** [comment]
I'd like to, but I'm missing time and knowledge
